specification/v3.5/OSDM-online-api-v3.5.0.yml

```diff
@@ -11963,6 +11963,41 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/VAT'
+        appliedCurrenyConversion:
+            $ref: '#/components/schemas/CurrencyConversion'
+          
+    CurrencyConversion:
+      description: | 
+        details on a currency conversion applied
+      required:  
+        - originalCurrency
+        - origionalAmount
+      properites:
+        description:
+          description: |
+            description of the conversion, e.g. rate, rounding rules, reference to rules
+          type: string
+        originalCurrency:
+          $ref: '#/components/schemas/Currency'
+        originalAamount:
+          description: |
+            amount in cents
+          type: integer
+          format: int32
+          nullable: false
+        scale:
```

Comment:
Suggestion - `scale` is applied to `amount`. When working with currency rates, also something called "base amount" is needed. It is the amount of one currency to be meaningful to convert to other. Usually it is one like:

1 EUR to CZK

But currencies with major inflation have different grades like 

100 JPY to CZK

Examples

- official Czech railways currency rates work only with non-inflated currencies (base is always 1) http://jzk.cd.cz/everyone/jzk-osobni/2025/jzk202501_en.html
- Czech National Bank has rates against currencies with base 100, 1000, ... https://www.cnb.cz/cs/financni-trhy/devizovy-trh/kurzy-devizoveho-trhu/kurzy-devizoveho-trhu/index.html?date=14.02.2025 https://www.cnb.cz/cs/financni-trhy/devizovy-trh/kurzy-ostatnich-men/kurzy-ostatnich-men/

Reply:
When you look at the Czeck National Bank rates the base amount is used to display the rate always with 3 fraction digits. 
The content is the same if you use base always as 1 and vary the number of digits in the rate.

Reply:
Yes, it is possible to work with `scale` only. `base` helps to make human-readable representation of that rate but it is not critical.